source/developers/cook-books/aws/upload-transcode-video.rst

```diff
@@ -0,0 +1,239 @@
+
+.. image:: /_static/images/guides/s3/mediaConvert-transcoded-form.png
+   :alt: AWS MediaConvert - Form Edit Transcoding Job Finished
+   :align: center
+
+|
+
+Here's how it will look like when we preview the page where we added the video:
+
+.. image:: /_static/images/guides/s3/mediaConvert-video-preview.png
+   :alt: AWS MediaConvert - Video Preview on Page
+   :align: center
+
+|
+
+---------------------------
+Step 5: Publish the changes
+---------------------------
+
+The next step is to publish the changes.  Remember to publish not just the page where we added the AWS MediaConvert data source,
+but also the ``article.ftl`` and the ``aws.xml`` files too.
+
+.. image:: /_static/images/guides/s3/attachments-publish.png
+   :alt: AWS MediaConvert - Publish Changes
+   :align: center
+
+|
+
+-------------------------------------------------------
+Step 6: Enable the remote assets controller in Delivery
+-------------------------------------------------------
+
+In order for the videos to be publicly accessible in Delivery, the remote assets controller needs
+to be enabled.  To do this, in your delivery, open the file ``rendering-context.xml`` under
+``apache-tomcat/shared/classes/crafter/engine/extension/`` and edit the file to contain the following:
+
+.. code-block:: xml
+   :linenos:
+
+       <?xml version="1.0" encoding="UTF-8"?>
+        <beans xmlns="http://www.springframework.org/schema/beans"
+               xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+               xmlns:util="http://www.springframework.org/schema/util"
+               xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd  http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd">
+
+   <import resource="classpath*:crafter/engine/mode/multi-tenant/simple/rendering-context.xml" />
+
+    <bean id="crafter.remoteAssetsRequestHandler" class="org.craftercms.engine.controller.RemoteAssetsRequestHandler"
+          init-method="init">
+     <property name="remoteFileResolver" ref="crafter.remoteFileResolver"/>
+     <property name="disableCaching" value="${crafter.engine.remoteAssets.disableCaching}"/>
+    </bean>
+
+    <util:map id="crafter.urlMappings">
+     <entry key="/api/**" value-ref="crafter.restScriptsController"/>
+     <entry key="/api/1/services/**" value-ref="crafter.restScriptsController"/> <!-- Deprecated mapping, might be removed in a later version -->
+     <entry key="/static-assets/**" value-ref="crafter.staticAssetsRequestHandler"/>
+     <entry key="/remote-assets/**" value-ref="crafter.remoteAssetsRequestHandler"/>
+     <entry key="/*" value-ref="crafter.pageRenderController"/>
+    </util:map>
+
+   </beans>
+
+|
+
+.. note::
+   Enabling the remote assets controller opens up all access to the files of the remote profiles configured in Studio
```

Comment:
Can we elaborate on what this means?

Reply:
@avasquez614 was the one who gave me the wording for the note... =D

Reply:
I think we need this elaborated. Happy to discuss further, reach out.